usr/share/rear/conf/default.conf

```diff
@@ -2767,6 +2767,10 @@ BLOCKCLONE_SOURCE_DEV=""
 BLOCKCLONE_PROG=""
 # Options that should be passed to BLOCKCLONE_PROG
 BLOCKCLONE_PROG_OPTS=""
+# Do we want to compress the image with gzip (.gz suffix), xz (.xz suffix), bzip2 (.bz2 suffix), zstd (.zst suffix)
+BLOCKCLONE_PROG_COMPRESS=""
```

Comment:
IMHO here should be a hint that compression only works with `dd` as it is only implemented for that.